### DISCUSSION TITLE
How to resize images in Github

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#27] [Show and tell]
To resize images in GitHub, use the HTML "**img**" tag with "**width**" and "**height**" attributes.
The "width" and "height" can be pixels or percentage and write both or only one attribute.

You can use the "githubusercontent" url that appears after attaching an image to the post.

From:
```
![image](https://user-images.githubusercontent.com/image.png)
```

To:
```html
<img src="https://user-images.githubusercontent.com/image.png" width="50%"/>
```
```html
<img src="https://user-images.githubusercontent.com/image.png" width="400"/>
```

---

An alternative is to organize the images with tables:
- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

**Comment by WolfgangZ42** [comment]
Just to try it ...
<img src="https://user-images.githubusercontent.com/121806238/215095348-2e24ad6d-5971-426f-854b-c59a503cb98b.png" width="200"/>
